src/Nevermind/Nevermind.Runner/Bootstraper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Nevermind.Core;
+using Nevermind.Json;
+using Nevermind.JsonRpc;
+using Nevermind.JsonRpc.Module;
+using Unity;
+
+namespace Nevermind.Runner
+{
+    public class Bootstraper
+    {
```

Comment:
I suggest we write our own simple container or use something different than Unity
https://github.com/danielpalme/IocPerformance